documentation/src/main/asciidoc/topics/marshalling.adoc

```diff
@@ -207,6 +209,13 @@ properties or specifying them in the {brandname} configuration.
 include::config_examples/deserialization_whitelist.xml[]
 ----
 
+[NOTE]
+====
+Java classes that you add to the deserialization whitelist apply to the
+{brandname} `CacheManager` interface and can deserialize to all caches that the
+`CacheManager` controls.
```

Comment:
I don't think we should say `CacheManager` interface, this doesn't actually exist and the white list is on the `EmbeddedCacheManager` interface. Instead we should say that the whitelist is applied to the `CacheContainer`, because this also applies when setting the white list via System properties on the server.